lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -474,7 +476,10 @@
                 success: this.clear_errors_then(function() {
                     return alert(fullSuccessMessage);  // eslint-disable-line no-alert
                 }),
-                error: statusAjaxError(function() {
+                error: statusAjaxError(function(response) {
+                    if (response.responseText === 'Specified module does not support rescoring.') {
```

Comment:
So in this case, given that we want to surface the message @srpearce outlined above as a product requirement, I should use that as the API-level error message and surface it directly from the error caught in the JS layer, right?